en/docs/Learn/Extensions/SAML2SSO/configuring-identity-server-as-idp-for-sso.md

```diff
@@ -47,7 +50,7 @@ For example, let's share a JDBC user store (MySQL) with both the WSO2 Identity S
 
 5.  Add the same datasource configuration above to `<IS_HOME>/repository/conf/deployment.toml` file.
 
-7.  The Identity Server has an embedded LDAP user store by default. As this is enabled by default. To change this to database user store, add the following to the `<IS_HOME>/repository/conf/datasources/deployment.toml`, follow the instructions in [Internal JDBC User Store Configuration](https://is.docs.wso2.com/en/5.9.0/learn/configuring-a-jdbc-user-store/) to disable the default LDAP and enable the JDBC user store instead.
+7.  The Identity Server has an embedded LDAP user store by default. As this is enabled by default. To change this to database user store, add the following to the `<IS_HOME>/repository/conf/deployment.toml`, follow the instructions in [Internal JDBC User Store Configuration](https://is.docs.wso2.com/en/next/setup/configuring-a-jdbc-user-store/) to disable the default LDAP and enable the JDBC user store instead.
```

Comment:
Since next is the URL of an unreleased version, we should be using the most recently released major version that APIM 3.0 supports

Reply:
Thanks. Fixed https://github.com/wso2/docs-apim/pull/544/commits/8931fbcc92eead2bd0dee32572193db7652507df